test/ctia/task/migration/store_test.clj

```diff
@@ -0,0 +1,304 @@
+        (is (apply < (map :modified (concat fetched-tool-asc))))
+        (is (apply > (map :modified (concat fetched-tool-desc))))
+        (is (apply < (map :modified (concat fetched-malware-asc))))
+        (is (apply > (map :modified (concat fetched-malware-desc)))))))
+
+  (es-index/delete! es-conn "test_*"))
+
+(deftest fetch-deletes-test
+  (whoami-helpers/set-whoami-response "45c1f5e3f05d0"
+                                      "foouser"
+                                      "foogroup"
+                                      "user")
+  (post-bulk examples)
+  (Thread/sleep 1000) ;; ensure indices refresh
+  (let [[sighting1 sighting2] (:parsed-body (helpers/get "ctia/sighting/search"
+                                                         :query-params {:limit 2 :query "*"}
+                                                         :headers {"Authorization" "45c1f5e3f05d0"}))
+        [tool1 tool2] (:parsed-body (helpers/get "ctia/tool/search"
+                                                 :query-params {:limit 2 :query "*"}
+                                                 :headers {"Authorization" "45c1f5e3f05d0"}))
+        [malware1] (:parsed-body (helpers/get "ctia/malware/search"
+                                              :query-params {:limit 1 :query "*"}
+                                              :headers {"Authorization" "45c1f5e3f05d0"}))
+        sighting1-id (-> sighting1 :id long-id->id :short-id)
+        sighting2-id (-> sighting2 :id long-id->id :short-id)
+        tool1-id (-> tool1 :id long-id->id :short-id)
+        tool2-id (-> tool2 :id long-id->id :short-id)
+        malware1-id (-> malware1 :id long-id->id :short-id)
+        _ (delete (format "ctia/sighting/%s" sighting1-id)
+                  :headers {"Authorization" "45c1f5e3f05d0"})
+        _ (Thread/sleep 1)
+        since (time/now)
+        _ (delete (format "ctia/sighting/%s" sighting2-id)
+                  :headers {"Authorization" "45c1f5e3f05d0"})
+        _ (delete (format "ctia/tool/%s" tool1-id)
+                  :headers {"Authorization" "45c1f5e3f05d0"})
+        _ (delete (format "ctia/tool/%s" tool2-id)
+                  :headers {"Authorization" "45c1f5e3f05d0"})
+        _ (delete (format "ctia/tool/%s" malware1-id)
+                  :headers {"Authorization" "45c1f5e3f05d0"})
+        {:keys [sighting tool malware]} (sut/fetch-deletes [:sighting :tool] since)]
+    (is (= #{(:id tool1) (:id tool2)} (set (map :id tool)))
+        "fetch-deletes shall return tool1 and tool2 that were deleted after since")
+    (is (= #{(:id sighting2)} (set (map :id sighting)))
+        "fetch-deletes shall return only entites that were deleted after since")
+    (is (nil? malware) "fetch-deletes shall only retrieve entity types given as parameter"))
+  (es-index/delete! es-conn "ctia_*"))
+
+
+(deftest init-get-migration-test
+  (post-bulk examples)
+  (Thread/sleep 1000) ; ensure indices refresh
```

Comment:
It could be better to use the `refresh?` param instead of a `Thread/sleep`

Reply:
there is no `refresh?` parameter on that function that directly request a local CTIA bulk route. I finally triggered manually the refresh.